### PR TITLE
Dmytro tymoshchenko tasks

### PR DESCRIPTION
**Pull request by pazuzu-ua**

### CODE REVIEW

File: tests/test_task_178b.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        self.assertEqual(expected_answer, answer)
+ 
+    def test_task178b_valid_empty(self):
+        """Test when one of args is zero result is zero."""
+        entry_sequence = []
+        expected_answer = 0
+        answer = task_178b(entry_sequence)
+        self.assertEqual(expected_answer, answer)
+
+    def test_task178b_valid_str(self):
```
Comment (Misha86) [inline tests/test_task_178b.py:26]: change docstring format
"""Docstring......
...........
"""

File: tests/test_task_178c.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    """Test for function test_178c"""
+
+    def test_task178v_valid(self):
+        """Test that task_178c function returns correct result."""
+        entry_sequence = (16, 9, 1, 14, 7, 22, 25, 121, 1, 0, 100)
+        expected_answer = 3
+        answer = task_178c(entry_sequence)
+        self.assertEqual(expected_answer, answer)
+
+    def test_task178v_negative(self):
```
Comment (Misha86) [inline tests/test_task_178c.py]: change docstring format
"""Docstring......
...........
"""

File: tests/test_task_178c.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+
+    def test_task178v_valid_empty(self):
+        """Test when there are no args result is zero."""
+        entry_sequence = []
+        expected_answer = 0
+        answer = task_178c(entry_sequence)
+        self.assertEqual(expected_answer, answer)
+
+    def test_task178v_valid_str(self):
```
Comment (Misha86) [inline tests/test_task_178c.py]: change docstring format
"""Docstring......
...........
"""

File: tests/test_task_554.py [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        self.assertEqual(expected_answer, answer)
+
+    def test_task554_valid_negative(self):
+        """Test when one of args is zero result is an empty list."""
+        entry_sequence = -8
+        expected_answer = []
+        answer = task_554(entry_sequence)
+        self.assertEqual(expected_answer, answer)
+
+    def test_task178v_valid_str(self):
```
Comment (Misha86) [inline tests/test_task_554.py]: change docstring format
"""Docstring......
...........
"""

File: tests/test_task_554.py [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        """Test when one of args is incorrect data type an
```
Comment (Misha86) [inline tests/test_task_554.py:34]: change docstring format
"""Docstring......
...........
"""

File: tests/test_task_554.py [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+        self.assertEqual(expected_answer, answer)
+
+    def test_task178v_valid_str(self):
+        """Test when one of args is incorrect data type an 
+        AssertionError exception is called."""
+        entry_sequence = ""
+        with self.assertRaises(AssertionError):
+            task_554(entry_sequence)
+
+    def test_task178v_valid_str(self):
```
Comment (Misha86) [inline tests/test_task_554.py]: change docstring format
"""Docstring......
...........
"""

**Review by Misha86 — COMMENTED** [review]
change docstring format
"""Docstring......
...........
"""

**Comment by datsiuk-ira** [comment]
in file 331a/b rename function from task_108 to task_331a and task_331b

**Pull request merged**